altmetric/explorer/api/test_api.py

```diff
@@ -0,0 +1,48 @@
+import pytest
+
+from . import Client
+from .env import API_KEY, API_SECRET
+
+
+@pytest.fixture
+def api_client():
+    return Client('https://www.altmetric.com/explorer/api', API_KEY, API_SECRET)
+
+
+def test_api_checks_for_invalid_key_and_secret():
+    with pytest.raises(ValueError):
+        Client(
+            'https://www.altmetric.com/explorer/api', None, None)
+
+
+@pytest.mark.parametrize("client_fn", [
```

Comment:
Is this how the Altmetric API responses are getting stubbed in the tests?

Reply:
The tests are currently actually doing real network calls.....

Reply:
This is a low-use python client, the tests are few and we don't have them running regularly in CI so should be fine but it might be worth stubbing out these calls just to doubly make sure